src/main/resources/lang/default.lang

```diff
@@ -1953,6 +1953,19 @@ quit reasons:
 	kicked: kicked
 	timed_out: timed out
 
+# -- Transform Reasons --
+transform reasons:
+	cured: zombie cure, zombie curing, zombie cured, cured
+	drowned: entity drowned, drowned, zombie converting to drown, zombie drown
+	frozen: entity froze, entity frozen, skeleton frozen, skeleton froze, skeleton converting to stray
+	lightning: lightning, creeper super charge, super charged creeper
```

Comment:
`lightning` doesn't compare properly when the cause is in a variable, but I don't know if this is very severe.
```
on witch transform:
    broadcast event-transformreason
    if event-transformreason is lightning:
        broadcast "lightning yay"
    set {_a} to event-transformreason
    if {_a} is lightning:
        broadcast "lightning yay 2"
```
![image](https://github.com/SkriptLang/Skript/assets/10354869/5ba48b3d-fbf7-473a-b042-2d81cf9fc815)

The other reasons should also be tested for conflicts, I haven't tested them all.

Reply:
I have a pending pull request for fixing literal comparison in variables.